src/routes/documents/[id]-[slug]/sidebar/Notes.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+  import type { Document, Note } from "$lib/api/types";
+
+  import { _ } from "svelte-i18n";
+  import { Note16, Note24 } from "svelte-octicons";
```

Comment:
How great that there's an existing Note icon! I thought it was a perfect fit when I saw it.